esphome/components/tm1637/tm1637.h

```diff
@@ -76,6 +77,7 @@ class TM1637Display : public PollingComponent {
   uint8_t intensity_;
   uint8_t length_;
   bool inverted_;
+  bool on_;
```

Comment:
If we don't initialize, the display will be off for everyone
```suggestion
  bool on_{true};
```